fvm/src/syscalls/network.rs

```diff
@@ -27,15 +17,18 @@ pub fn total_fil_circ_supply(context: Context<'_, impl Kernel>) -> Result<sys::T
 
 pub fn context(context: Context<'_, impl Kernel>) -> crate::kernel::Result<SyscallNetworkContext> {
     Ok(SyscallNetworkContext {
-        network_curr_epoch: context.kernel.network_epoch(),
```

Comment:
the function name is only called `network_epoch` because it's on the `NetworkOps` trait.